IFC4x3/Sections/Resource definition data schemas/Schemas/IfcGeometricConstraintResource/Functions/IfcCorrectLocalPlacement/Documentation.md

```diff
@@ -1,3 +1,3 @@
 The function checks that a relative placement (i.e. relative to another local placement, and not grid placement) of a 3D local placement has to be relative to a 3D parent placement (and not to a 2D parent placement).
```

Comment:
```suggestion
The function checks that a relative placement (i.e. relative to another local placement, and not grid placement or a linear placement) of a 3D local placement has to be relative to a 3D parent placement (and not to a 2D parent placement).
```

Reply:
Thanks for the tip but it is actually the other way around. A local placement should be able to reference a linear placement.